stable/nfs-client-provisioner/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: 2.1.1
+appVersion: 3.0.1
```

Comment:
why does the appVersion need to be changed in this PR?

Reply:
Because it was wrong in the previous, we're installing 3.0.1 ( image tag )